lib/builders/dataTypes/datePickerDataTypeBuilder.ts

```diff
@@ -6,8 +6,8 @@ export class DatePickerDataTypeBuilder extends DataTypeBuilder {
 
   constructor() {
     super();
-    this.propertyEditorAlias = "Umbraco.DateTime";
-    this.propertyEditorUiAlias = "";
+    this.editorAlias = "Umbraco.DateTime";
+    this.editorUiAlias = "Umb.PropertyEditorUi.DatePicker";
   }
 
   toOffsetTime(offsetTime: boolean) {
```

Comment:
To ensure consistency, I suggest that you change all the setter methods to use the 'with' prefix. For example, this method should be "withOffsetTime" instead of "toOffsetTime". What do you think?

Reply:
I agree with you. should be consistent! 😄